shared/structures/src/PermissionRole.ts

```diff
@@ -177,6 +177,19 @@ export class PermissionRoleDetailed extends PermissionRole {
         return this.getResourcePermissions(type, id)?.hasAccessRight(right) ?? false;
     }
 
+    hasAccessRightForSomeResource(type: PermissionsResourceType, right: AccessRight): boolean {
+        if (this.hasAccessRight(right)) {
+            return true;
+        }
+
+        const resource = this.resources.get(type);
+        if (!resource) {
+            return false;
+        }
+
+        return resource.size > 0;
```

Comment:
Wat is het nut van de `right` property behalve in de global check? Lijkt me foutief

Reply:
Inderdaad